largescaleobserved.qmd

```diff
@@ -14,16 +14,10 @@ using AlgebraOfGraphics
 using Arrow
 using CairoMakie
 using CategoricalArrays
-using DataFrameMacros
 using DataFrames
-using GLM
-if contains(first(Sys.cpu_info()).model, "Intel")
-  using MKL
-end
 using MixedModels
-using MixedModelsMakie
-using ProgressMeter
-using SparseArrays
+#using ProgressMeter
```

Comment:
did you want all this commented out?

Reply:
I was trying to trim packages that were included in `using` statements but not actually used.  That is a task that is probably best left until after the content and text have stabilized.  For the time being I plan to comment out the `using` statements for packages because that is an action that can easily be reversed.